src/GoTrueClient.ts

```diff
@@ -1756,11 +1748,7 @@ export default class GoTrueClient {
   private async _removeSession() {
     this._debug('#_removeSession()')
 
-    if (this.persistSession) {
-      await removeItemAsync(this.storage, this.storageKey)
-    } else {
```

Comment:
Ah okay was slow to figure it out so calling it out explicitly for future ref feel free to correct me if my understanding is off.

Looks like we no longer need to check `persistSession` since there's a default `inMemorySession` when `persistSession` is set to false. Consequently there's no need to check for the `persistSession` before calling `removeItemAsync` since it will now default to the in memory session